### PR TITLE
docs: Updated commands.md and created events.md

### PR DESCRIPTION
**Pull request by lokerhp**
This is the Pull Request for Issue #12 
I hope I did everything correct 🙃 

If there are issues please let me know and I will try to fix them.

Greetings,
lokerhp

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Table of contents
 
 * [Introduction](../README.md)
+* [Installation](./installation.md)
```
Comment (huenique) [inline docs/README.md:4]: please remove this line as `installation.md` file doesn't exist anymore. thanks

**Review by huenique — CHANGES_REQUESTED** [review]
In `commands.md`, we've moved to using only double hash sign (##) for command section titles

**Comment by lokerhp** [comment]
I changed the things, Not sure why I added the extra #, but they are gone now.

**Review by huenique — APPROVED** [review]
looks good. merging now 👍

**Pull request merged**